install.py

```diff
@@ -103,52 +100,57 @@ def copy_completion_scripts(dest_dir):
     return None
 
 
+def install_rez_from_source(dest_dir):
+    _, py_executable = get_venv_executable(dest_dir, "python")
+    _, pip_executable = get_venv_executable(dest_dir, "pip")
+
+    # build wheel
+    run_command([py_executable, "setup.py", "bdist_wheel"])
```

Comment:
I think we can simplify and solidify this by doing:

```python
run_command([py_executable, "-m", "pip", "install", "."])
```
1. Using `py_executable -m pip` you don't need to take care of finding pip. Python will do it for us.
2. `pip install .` will build the wheel for you and install it for you. It is a little cleaner and a preferred approach from what I know. The other benefit is that you don't have to look for the wheel in the `dist` directory which could have some development stuff in there is not cleaned-up before running the `install.py` file.

Reply:
nice, changes coming